src/cmd/tag.go

```diff
@@ -0,0 +1,139 @@
+package cmd
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/opslevel/cli/common"
+	"github.com/opslevel/opslevel-go/v2023"
+	"golang.org/x/exp/slices"
+
+	"github.com/spf13/cobra"
+)
+
+var resourceType string
+
+var createTagCmd = &cobra.Command{
+	Use:   "tag --type=RESOURCE_TYPE [--assign] RESOURCE_ID KEY VALUE",
+	Short: "Create/assign a tag",
+	Long:  "Create/assign a tag",
+	Example: `
+opslevel create tag --type=Team ID|ALIAS KEY VALUE
+`,
+	Args:       cobra.ExactArgs(3),
+	ArgAliases: []string{"RESOURCE_ID", "KEY", "VALUE"},
+	Run: func(cmd *cobra.Command, args []string) {
+		if resourceType == "" {
```

Comment:
Can we consolidate these two `if` statements into a function like `func validateResourceTypeArg()` below?
This will make the command easier to test down the road 😃

Reply:
added!